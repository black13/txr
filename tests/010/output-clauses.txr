@(bind l ("zero" "one" "two" "three" "four"))
@(bind a (@[l 0..0] @[l 0..2] @[l 0..3] @[l 0..4] @[l 0..5]))
@(output)
@(repeat :counter i)
counter:@i:A
@(repeat)
  main:@a
@(first)
  first:@a
@(mod 1 2)
  odd:@a
@(mod 0 2)
  even:@a
@(last)
  last:@a
@(modlast 1 2)
  oddlast:@a
@(modlast 0 2)
  evenlast:@a
@(empty)
  empty
@(end)
counter:@i:B
@(repeat)
  main:@a
@(first)
  first:@a
@(mod 1 2)
  odd:@a
@(mod 0 2)
  even:@a
@(last)
  last:@a
@(empty)
  empty
@(end)
@(end)
@(end)
