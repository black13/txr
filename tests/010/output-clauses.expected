counter:0:A
  empty
counter:0:B
  empty
counter:1:A
  first:zero
  oddlast:one
counter:1:B
  first:zero
  last:one
counter:2:A
  first:zero
  odd:one
  evenlast:two
counter:2:B
  first:zero
  odd:one
  last:two
counter:3:A
  first:zero
  odd:one
  even:two
  oddlast:three
counter:3:B
  first:zero
  odd:one
  even:two
  last:three
counter:4:A
  first:zero
  odd:one
  even:two
  odd:three
  evenlast:four
counter:4:B
  first:zero
  odd:one
  even:two
  odd:three
  last:four
